src/database/migrations/20240321091346-platform-config-role-permission.js

```diff
@@ -0,0 +1,112 @@
+'use strict'
+
+require('module-alias/register')
+const userRequests = require('@requests/user')
+require('dotenv').config()
+const common = require('@constants/common')
+const Permissions = require('@database/models/index').Permission
+const RolePermissionMapping = require('@database/models/index').RolePermission
+
+const getPermissionId = async (module, request_type, api_path) => {
+	try {
+		const permission = await Permissions.findOne({
+			where: { module, request_type, api_path },
+		})
+		if (!permission) {
+			throw error
+		}
+		return permission.id
+	} catch (error) {
+		throw error
+	}
+}
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+	async up(queryInterface, Sequelize) {
+		try {
+			const rowToDelete = await RolePermissionMapping.findOne({
+				where: {
+					role_title: common.PUBLIC_ROLE,
+					permission_id: await getPermissionId('platform', ['GET'], '/mentoring/v1/platform/config'),
+					module: 'platform',
+					request_type: ['GET'],
+					api_path: '/mentoring/v1/platform/config',
+				},
+			})
+			await rowToDelete.destroy()
+		} catch (error) {
+			console.error('Error deleting row:', error)
+		}
+		try {
+			const rolePermissionsData = [
+				{
+					role_title: common.MENTEE_ROLE,
```

Comment:
@sumanvpacewisdom we don't have ALL role?, if we introduce new role, again we need to migrations for that